content/enterprise_influxdb/v1.9/administration/manage/security/introduction-to-auth.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
After reading through this doc, I'd lean toward removing the doc and keeping the Choose one of the following three options on the main "Manage user and permissions" doc

Reply:
This document contains subtle technical distinctions and important context that we don't have anywhere else.